src/os/flatpak.zig

```diff
@@ -135,6 +148,77 @@ pub const FlatpakHostCommand = struct {
         }
     }
 
+    /// Wait for the process to end asynchronously via libxev. This
+    /// can only be called ONCE.
+    pub fn wait_xev(
```

Comment:
Functions in Zig are camelCased

```suggestion
    pub fn waitXev(
```